search.php

```diff
@@ -30,37 +33,24 @@
 	$context['selected_sort'] = $selected_sort;
 }
 
-switch ( $context['selected_sort'] ) {
-	case 'recent':
-		$context['posts'] = Timber::get_posts( [
-			// TODO - Find solution for Timber bug (see https://github.com/timber/timber/issues/935).
-			// which does not include attachments to get_posts() results if we supply an array as an argument.
-			// This might be related to Timber not collaborating well with SearchWP. Needs further investigation.
-			's' => $search,
-			'post_type' => [
-				'post',
-				'page',
-				'attachment',
-			],
-			'post_status' => 'any',
-			'orderby' => 'post_date',
-			'order' => 'DESC',
-			'numberposts' => -1,
-		] );
-		break;
-	default:
-		// TODO - Add 'numberposts' option.
-		// The issue seems like it is related with Timber and SearchWP not collaborating very well here.
-		// If we add a parameter to Timber::get_posts() then it looks like it skips SearcWP and does
-		// not include attachments. So, for now I leave the default sort without parameter to
-		// be able to review the attachment searching functionality.
-		$context['posts'] = Timber::get_posts();
-}
-// Cast to array for forward compatibility with php 7.2 which requires count parameter to be array or object that implements Countable.
-$found_posts = count( (array) $context['posts'] );
+/*
+  The issue seems like it is related with Timber and SearchWP not collaborating very well here. After investigating this we found that
+  If we do not pass an argument to Timber::get_posts() then it falls back to the main WP_Query which works with SearchWP.
+  If we pass an argument to Timber::get_posts() then it creates a subquery and SearchWP is not aware of that and therefore we do not get attachemnts included in search results.
+  A solution is to proceed without passing query options to get_posts at all
+  and instead use the `edit_searchwp_query_orderby` of searchwp to edit the order by part of the main WP_Query directly.
+*/
+$context['posts'] = Timber::get_posts();
```

Comment:
I think for this description we can just leave instructions not to insert query args directly into this call and that get_posts uses the main query if no params are passed in. The solution we came out with here is more technically correct because we don't introduce unnecessary queries on the page load and instead modify the main query.

```
/* With no args passed to this call, Timber uses the main query which we filter for customisations via 
 * P4_Master_Site class.
 *
 * When customising this query, use filters on the main query to avoid bypassing SearchWP's 
 * handling of the query
 */